internal/tools/sports/tool_test.go

```
@@ -0,0 +1,202 @@
+
+	raw, err := handlers[sports.ReservationCreateToolID](ctx, request, json.RawMessage(`{"venue_id":"venue-gym","project_id":"project-badminton","slot_id":"slot-morning","count":2}`))
+	if err != nil {
+		t.Fatal(err)
+	}
+	var created sports.ReservationResult
+	if err := json.Unmarshal(raw, &created); err != nil || created.Reservation.Status != sports.StatusConfirmed {
+		t.Fatalf("created=%s err=%v", raw, err)
+	}
+
+	payload, _ := json.Marshal(map[string]string{"reservation_id": created.Reservation.ID})
+	raw, err = handlers[sports.ReservationCancelToolID](ctx, request, payload)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to raw

(ineffassign)